<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calDialog</class>
 <widget class="QDialog" name="calDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Clear">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>99</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
    </font>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNum1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>128</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>128</y>
     <width>32</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>+</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>/</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditNum2">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>128</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditResult">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>129</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>53</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>53</width>
     <height>18</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
    </font>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
